packages/components/addon/components/hds/dropdown/list-item/copy-item.js

```diff
@@ -38,14 +38,16 @@ export default class HdsDropdownListItemCopyItemComponent extends Component {
 
   @action
   async copyCode() {
-    // https://developer.mozilla.org/en-US/docs/Web/API/Clipboard
-    await navigator.clipboard.writeText(this.args.text);
-    const result = await navigator.clipboard.readText();
+    let copyText = await this.args.text;
```

Comment:
i don't think this is asynchronous

```suggestion
    let copyText = this.args.text;
```